app/lib/pdf_filler/nj1040_pdf.rb

```diff
@@ -104,7 +104,46 @@ def hash_for_pdf
         answers.merge!(dependent_hash)
       end
 
-      # lines 13 and 30
+      # line 10
+      if @xml_document.at("Header NumOfQualiDependChild")
+        qualifying_children_count = @xml_document.at("Header NumOfQualiDependChild").text.to_i
+        answers.merge!(
+          insert_digits_into_fields(
+            qualifying_children_count,
+            [ "undefined_12", "Text47" ],
+            as_decimal: false
+          )
+        )
+        answers.merge!({ 'x  1500': calculated_fields.fetch(:NJ1040_LINE_10_EXEMPTION) })
+      end
+
+      # line 11
+      if @xml_document.at("Header NumOfOtherDepend")
+        other_dependent_count = @xml_document.at("Header NumOfOtherDepend").text.to_i
+        answers.merge!(
+          insert_digits_into_fields(
+            other_dependent_count,
+            [ "undefined_13", "Text48" ],
+            as_decimal: false
+          )
+        )
+        answers.merge!({ 'x  1500_2': calculated_fields.fetch(:NJ1040_LINE_11_EXEMPTION) })
+      end
+
+      # line 12
+      if @xml_document.at("Exemptions DependAttendCollege")
+        count = @xml_document.at("Exemptions DependAttendCollege").text.to_i
+        answers.merge!(
+          insert_digits_into_fields(
+            count,
+            [ "undefined_14", "Text49" ],
+            as_decimal: false
+          )
+        )
+        answers[:'x  1000_4'] = count * 1_000
+      end
```

Comment:
i moved this from below in the file to get the lines a little bit more in order (they aren't throughout the file but every little bit helps). And I also `insert_digits_into_fields` now supports `as_decimal: false` so i refactored this as discussed on [the PR for line 12](https://github.com/codeforamerica/vita-min/pull/5011#discussion_r1852421036)